.github/workflows/github_actions.yml

```diff
@@ -52,5 +52,7 @@ jobs:
       - name: checkout code
         uses: actions/checkout@v2
       - name: coding convention
-        run: sh .ci/check-format.sh
+        run: |
```

Comment:
I am no expert of YAML. What is the pipe `I` used for?

Reply:
In `Github Actions`, it means the start of the multiple commands.

If `|` is not used, we can only use a single command or wrap multiple commands into a single script and then invoke it.

Reply:
Learnt!